lighthouse-core/lib/traces/pwmetrics-events.js

```diff
@@ -131,39 +131,15 @@ class Metrics {
         }
       },
       {
-        name: 'Time to Interactive (vAlpha)',
```

Comment:
let's keep TTI in here for now. 
i'd like them to live side-by-side for a while for folks to get comfortable.
then we can drop.